modules/pruning-images.adoc

```diff
@@ -5,107 +5,89 @@
-to clear the cache after pruning. This can be accomplished by redeploying the registry:
+. Verify the custom resource definition contains the following `spec` and `status` fields:
 
+[source,yaml]
 ----
-$ oc rollout restart deployment/image-registry -n openshift-image-registry
+spec:
+  schedule: "*/1 * * * *" <1>
+  suspend: false <2>
+  keepTagRevisions: 3 <3>
+  keepYoungerThan: 60m <4>
+  resources: {} <5>
+  affinity: {} <6>
+  nodeSelector: {} <7>
+  tolerations: {} <8>
+  startingDeadlineSeconds: 60 <9>
+  history: <10>
+    successfulJobsHistoryLimit: 3
+    failedJobsHistoryLimit: 3
+status:
+  observedGeneration: 2 <11>
+  conditions: <12>
+  - type: Available
+    status: "True"
+    lastTransitionTime: 2019-10-09T03:13:45
+    reason: Ready
+    message: "Periodic image pruner has been created."
+  - type: Scheduled
+    status: "True"
+    lastTransitionTime: 2019-10-09T03:13:45
+    reason: Scheduled
+    message: "Image pruner job has been scheduled."
+  - type: Failed
+    staus: "False"
+    lastTransitionTime: 2019-10-09T03:13:45
+    reason: Succeeded
+    message: "Most recent image pruning job succeeded."
 ----
+<1> `schedule`: `CronJob` formatted schedule, defaults to daily at midnight for new clusters. This is a required field.
+<2> `suspend`: If set to `true`, the `CronJob` running pruning is suspended. This is an optional field, and it defaults to `false`.
+<3> `keepTagRevisions`: The number of revisions per tag to keep. This is an optional field, and it defaults to `3` if not set.
+<4> `keepYoungerThan`: Retain images younger than this duration. This is an optional field, and it defaults `60m` if not set.
+<5> `resources`: Standard Pod resource requests and limits. This is an optional field.
+<6> `affinity`: Standard Pod affinity. This is an optional field.
+<7> `nodeSelector`: Standard Pod node selector. This is an optional field.
+<8> `tolerations`: Standard Pod tolerations. This is an optional field.
+<9> `startingDeadlineSeconds`: Start deadline for `CronJob`. This is an optional field.
+<10> `history`: Sets the job history to be retained on the cluster. This object is optional.
+* `successfulJobsHistoryLimit`: The maximum number of successful jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
+* `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
+<11> `observedGeneration`: The generation observed by the operator.
+<12> `conditions`: The standard condition objects with the following types:
+ * `Available`: Indicates if the pruning job has been created. Reasons can be Ready or Error.
+* `Scheduled`: Indicates if the next pruning job has been scheduled. Reasons can be Scheduled, Suspended, or Error.
+* `Failed`: Indicates if the most recent pruning job failed.
 
-If the integrated registry uses a Redis cache, you must clean the database
```

Comment:
I removed this content because it didn't immediately look like it was still relevant. Please let me know if it does actually belong in the auto pruning documentation.